### PR TITLE
Fix sample deployment issue for samples except virtual fire alarm

### PR DESCRIPTION
**Pull request by charithag**

### CODE REVIEW

File: modules/samples/sample_pom.xml [hunk 1]
```diff
@@ -401,75 +407,84 @@
...
-				<systemPath>${basedir}/../../../../repository/components/plugins/com.google.gson_2.3.1.jar</systemPath>
 			</dependency>
 
+			<!--Dependencies on XMPP Client Library-->
+			<dependency>
+				<groupId>org.igniterealtime.smack.wso2</groupId>
+				<artifactId>smack</artifactId>
+				<version>${smack.wso2.version}</version>
+                <scope>system</scope>
+                <systemPath>${basedir}/../../../../repository/components/plugins/smack_3.0.4.wso2v1.jar</systemPath>
```
Comment (DilanUA) [inline modules/samples/sample_pom.xml:418]: fix formatting issues.

File: modules/samples/sample_pom.xml [hunk 2]
```diff
@@ -401,75 +407,84 @@
...
+			</dependency>
+			<dependency>
+				<groupId>org.igniterealtime.smack.wso2</groupId>
+				<artifactId>smackx</artifactId>
+				<version>${smackx.wso2.version}</version>
+                <scope>system</scope>
```
Comment (DilanUA) [inline modules/samples/sample_pom.xml:424]: fix formatting issues.

File: modules/samples/sample_pom.xml [hunk 3]
```diff
@@ -401,75 +407,84 @@
...
-		<eclipse.equinox.common.version>3.6.100.v20120522-1841</eclipse.equinox.common.version>
-		<eclipse.osgi.version>3.8.1.v20120830-144521</eclipse.osgi.version>
-
-		<!--Orbit versions-->
-		<orbit.h2.engine.version>1.2.140.wso2v3</orbit.h2.engine.version>
-		<orbit.tomcat.version>7.0.52.wso2v5</orbit.tomcat.version>
-		<orbit.tomcat.jdbc.pooling.version>7.0.34.wso2v2</orbit.tomcat.jdbc.pooling.version>
-
-		<eclipse.paho.version>0.4.0</eclipse.paho.version>
+	</dependencyManagement>
```
Comment (DilanUA) [inline modules/samples/sample_pom.xml:430]: fix formatting issues.

File: modules/samples/sample_pom.xml [hunk 4]
```diff
@@ -401,75 +407,84 @@
...
 
+        <!-- Source code -->
+        <wso2.maven.compiler.source>1.7</wso2.maven.compiler.source>
+        <wso2.maven.compiler.target>1.7</wso2.maven.compiler.target>
+        <carbon.analytics.version>1.0.3</carbon.analytics.version>
 
+        <!--XMPP/MQTT Version-->
+        <smack.wso2.version>3.0.4.wso2v1</smack.wso2.version>
+        <smackx.wso2.version>3.0.4.wso2v1</smackx.wso2.version>
+        <paho.mqtt.version>0.4.0</paho.mqtt.version>
```
Comment (DilanUA) [inline modules/samples/sample_pom.xml:486]: unnecessary new line.

File: modules/samples/sample_pom.xml (hunk 1 above)
Comment (charithag) [inline modules/samples/sample_pom.xml:418]: Can't fix as this code is obsolete and no longer exists.

File: modules/samples/sample_pom.xml (hunk 2 above)
Comment (charithag) [inline modules/samples/sample_pom.xml:424]: Can't fix as this code is obsolete and no longer exists.

File: modules/samples/sample_pom.xml (hunk 3 above)
Comment (charithag) [inline modules/samples/sample_pom.xml:430]: Can't fix as this code is obsolete and no longer exists.

File: modules/samples/sample_pom.xml (hunk 4 above)
Comment (charithag) [inline modules/samples/sample_pom.xml:486]: Can't fix as this code is obsolete and no longer exists.

**Pull request merged**